source/data_processing.py

```diff
@@ -11,7 +11,7 @@
 
 
 def text_processing(data):
-    data = data.apply(lambda tweet: tweet.lower())  # lower register
+    data = data.str.lower()  # lower register
```

Comment:
It's is not what I've suggested.  Does it even work?

Reply:
Yes, it works